presto-main/src/main/java/com/facebook/presto/connector/system/ResourceGroupInfoSystemTable.java

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
On your commit message, I would omit the reasoning for why something is useful for users. That belongs in the PR or release notes as context for people doing a review. It does not help the historical record. If the commit helped with another commit or PR, that would be a different matter.